scripts/exregional_integration_test.py

```diff
@@ -89,46 +94,44 @@ def setup_logging(debug=False):
         help="Print debug messages.",
         required=False,
     )
-    parser.add_argument('unittest_args', nargs='*')
+    parser.add_argument("unittest_args", nargs="*")
     args = parser.parse_args()
     sys.argv[1:] = args.unittest_args
-   
-    fcst_dir = str(args.fcst_dir) 
+
+    FCST_DIR = str(args.fcst_dir)
     fcst_len = int(args.fcst_len)
     fcst_inc = int(args.fcst_inc)
```

Comment:
I'm curious why `FCST_DIR` (and `MODEL_CONFIGURE_FP`) got uppercase treatment here vs other variables.

Also, and I'm not asking for a change, but how about using the [`type=` parameter](https://docs.python.org/3/library/argparse.html#type) to `add_argument()` to do the conversion to `int` for `fcst_len` and `fcst_inc` instead of these statements, and then just use the already-pretty-concise `args.fcst_dir`, `args.fcst_len`, and `args.fcst_inc` values below after removing these three lines?

Reply:
I can make that move for the parameters, for sure. 

The all caps were suggestions from the linter. I just went with it.

Reply:
Actually, I think it's because FCST_DIR is a parameter, while the others are used as variables (their values change later).

Reply:
I wonder whether, if we just use `args.fcst_dir`, it becomes moot.

Reply:
It does, except for the one instance of `MODEL_CONFIGURE_FP`. But `FCST_DIR` goes away.